lib/pmdtester/runner.rb

```diff
@@ -58,15 +61,19 @@ def run_online_mode
 
       baseline_path = download_baseline(@options.base_branch)
 
-      # patch branch build pmd reports with same configuration as base branch
-      config_path = "#{baseline_path}/config.xml"
+      if @options.auto_config_flag
```

Comment:
Only online mode support auto-gen-config option now, maybe we should also extend local mode.

Reply:
:+1: sounds good